what

```diff
@@ -1,10 +1,15 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
```

Comment:
Could you actually use `python` in the shebang? That way it will work regardless of whether python2 or python3 is installed.